accessibility/accessibility-in-emails.md

```diff
@@ -0,0 +1,63 @@
+# Accessibility in emails
+
+This info sits alongside our general guide to writing emails at ????.
+
+## Intro
+
+Accessibility in emails can be summarised with 3 key points:
+
+1. Accessible emails allow you to reach a wider audience.
+1. Writing and visual design are key components of an accessible email campaign.
+1. Email templates can be made immediately more accessible with a few simple coding techniques.
+
+Creating accessible emails are not fundamentally different from creating accessible websites.
+
+Here are the few differing constraints and techniques that should be considered.
+
+## Layout
+
+- Rich visual emails typically use tables for layout.
+- This is because email clients use old and conservative browser engines that do not cope well nor consistently with CSS-based layouts.
+- Using ARIA to set `role="presentation"` on layout tables will help prevent them being seen as data tables by screen readers.
+
+## Specify a language
+
+- Setting the language of the email on the root element will allow assistive technology to use the appropriate language profile.
+
+## Alternative text
+
+- Marketing emails tend to make heavy use of images.
```

Comment:
By the time I was reading this, I've already forgotten that this was a list of techniques _and constraints_ so I thought "is this a good thing?"

It may make more sense to have the constraint as an introduction to why whe use those techniques, by simply removing it from the list, e.g.

```
## Alternative text

Marketing emails tend to make heavy use of images.

- Alt text should be used as it would be on a web site
- [...]
```

Reply:
I like this idea. I've updated the sections to be constraint first, and then some solutions afterwards.